website/api/controllers/deliver-talk-to-us-form-submission.js

```
@@ -94,58 +94,106 @@ module.exports = {
       });
+
+      let bookingUrlByUserId = {
+        '005UG000006YYDVYA4': 'https://calendly.com/d/d3fs-28g-vdk/talk-to-us', //newYorkCity
+        '0054x0000086sOlAAI': 'https://calendly.com/d/dzyz-tt7-yt8/talk-to-us', //washingtonDc
+        '005UG000008y0wbYAA': 'https://calendly.com/d/ds9c-9vt-mz6/talk-to-us', //losAngeles
+        '0054x0000086wsGAAQ': 'https://calendly.com/d/dz4c-mjx-6xv/talk-to-us', //sanFrancisco
+        '005UG000009NnSfYAK': 'https://calendly.com/d/ds88-n2m-ddt/talk-to-us', //stockholm
+      };
+
+      let eventUrlForThisUsersTerritory = bookingUrlByUserId[territoryUserId];
+      if(!eventUrlForThisUsersTerirtory) {
```

ESLint

[error] 'eventUrlForThisUsersTerirtory' is not defined.

(no-undef)